judgments/courts.py

```diff
@@ -0,0 +1,104 @@
+courts = [
```

Comment:
Could we move this file into a subdirectory, called something like `fixtures`? Just to distinguish it from business-logic code that sits at the top-level. (You might need to make the subdirectory into a python module by add an empty `__init__.py` file)